src/include/daos_obj.h

```diff
@@ -127,6 +127,12 @@ enum {
 	DAOS_COND_AKEY_FETCH	= (1 << 6),
 	/** Mask for convenience */
 	DAOS_COND_MASK		= ((1 << IO_FLAGS_COND_BITS) - 1),
+
+	/**
+	 * Not copy application buffer when cache modification on client
+	 * for DTX.
+	 */
+	DAOS_ZERO_COPY		= (1ULL << 63)
```

Comment:
I'm still very much against this Not being the default option.

Reply:
You said that you would ask Argonne guys about there use cases, then to decide whether enable zero copy or not by default. So what is their suggestion?

Reply:
I have discussed with Johann, but we have no conclusion

Reply:
no conclusion about the use cases or no conclusion about the default behavior?

Reply:
No issue to be consider is that how can we clearly described which kind of buffers cannot be reused before TX committed.